albumentations/augmentations/transforms.py

```diff
@@ -2572,6 +2572,42 @@ def get_transform_init_args_names(self):
         return ()
 
 
+class Downscale(ImageOnlyTransform):
+    """Decreases image quality by downscaling and upscaling back.
+
+    Args:
+        scale_min (float): lower bound on the image scale. Should be < 1.
+        scale_max (float):  lower bound on the image scale. Should be .
+        interpolation: cv2 interpolation method. cv2.INTER_NEAREST by default
+
+    Targets:
+        image
+
+    Image types:
+        uint8, float32
+    """
+
+    def __init__(self, scale_min=.25, scale_max=.25, interpolation=cv2.INTER_NEAREST,
+                 always_apply=False, p=0.5):
+        super().__init__(always_apply, p)
+        assert scale_min <= scale_max, \
+            "Expected scale_min be less or equal scale_max, got {} {}".format(scale_min, scale_max)
+        assert scale_max < 1, "Expected scale_max to be less than 1, got {}".format(scale_max)
+        self.scale_min = scale_min
+        self.scale_max = scale_max
+        self.interpolation = interpolation
+
+    def apply(self, image, scale, interpolation, **params):
+        return F.downscale(image, scale=scale, interpolation=interpolation)
+
+    def get_params(self):
+        return {'scale': np.random.rand() * (self.scale_max - self.scale_min) + self.scale_min,
```

Comment:
You can use `random.uniform(self.scale_min, self.scale_max)`

Reply:
thnx, fixed